operator/v1/types_etcd.go

```diff
@@ -43,6 +43,16 @@ type EtcdSpec struct {
 	// +openshift:enable:FeatureGate=HardwareSpeed
 	// +optional
 	HardwareSpeed ControlPlaneHardwareSpeed `json:"controlPlaneHardwareSpeed"`
+
+	// BackendQuotaGiB sets the etcd backend storage size limit in gibibytes. Defaults to 8GiB.
```

Comment:
Just to add a little more colour to the explanation. Also, need to talk about the value being an int
```suggestion
	// backendQuotaGiB sets the etcd backend storage size limit in gibibytes.
	// The value should be an integer not less than 8 and not more than 32.
	// When not specified, the default value is 8.
```

Reply:
fixed 👍🏽